src/platform/forms-system/test/js/review/ReviewCollapsibleChapter.unit.spec.jsx

```diff
@@ -1104,6 +1104,32 @@ describe('<ReviewCollapsibleChapter>', () => {
       expect(queryByTestId('custom-page-review')).not.to.exist;
     });
 
+    it('should include noop navigation functions when rendering CustomPage in edit mode', () => {
+      const { pages, chapterKey, chapter, form } = getProps();
+      let result;
+      const CustomPage = props => {
+        result = props;
+        return <div data-testid="custom-page" />;
+      };
+      form.pages.test.editMode = true;
+      form.pages.test.CustomPage = CustomPage;
+      pages[0].CustomPage = CustomPage;
+      render(
+        <ReviewCollapsibleChapter
+          viewedPages={new Set()}
+          expandedPages={pages}
+          chapterKey={chapterKey}
+          chapterFormConfig={chapter}
+          form={form}
+          open
+        />,
+      );
+
+      expect(result.goBack.toString()).to.contain('noop()');
+      expect(result.goForward.toString()).to.contain('noop()');
+      expect(result.goToPath.toString()).to.contain('noop()');
```

Comment:
Checking for the function string containing `noop()` because the testing library adds some extra code inside. The original `function noop() {}` becomes: 

```js
function noop() {\n          cov_26h19nyxg8().f[17]++;\n        }
```

<img width="807" alt="empty function in unit tests" src="https://github.com/department-of-veterans-affairs/vets-website/assets/136959/e4e8bb49-ab34-426d-afa0-b8dd16c3f720">